testdata/project-v3-with-kustomize-v2/PROJECT

```diff
@@ -37,6 +37,7 @@ resources:
   group: crew
   kind: Admiral
   path: sigs.k8s.io/kubebuilder/testdata/project-v3-with-kustomize-v2/api/v1
+  plural: admirales
```

Comment:
```suggestion
  plural: admirals
```

I would standardize for the correct English term here, so `admirals` instead of `admirales`. Or is there a reason for doing the opposite when streamlining this?

Reply:
We have an option command to change the plurals which are required in some very specific cases
The goal of this PR is we only have the samples:
- project-v3
AND
- project-v3-with-kustomize-v2-alpha plugin

Doing exactly the same scaffold. So that we can:
a) Better evaluate the need or not to have the kustomize-v2 alpha plugin
b) If still required in a follow-up (after the next release) we will replace it with the go/v4-alpha plugin
v) So that people can easily compare and check what changed from go/v3 to go/v4-alpha so far.

Reply:
@erikgb

Reply:
I understand the need to streamline the scaffold, but what I don't understand is why you cannot standardize on `admirals` instead of `admirales`.

Reply:
That is a sample, we scaffold the project v3 using the option to ensure that we will not break the feature. 
The goal of this PR is only to ensure that project v3 sample is == project v3 with new kustomize.
We are not change nothing in the scaffold that is only "I want the sample testdata generate with the new plugin done using the same options used to build the project v3 scaffold so that we can diff and check what was changed from one to another"

If you would like to know why we have a flag `--plural:` that allows we change the plurals, see : https://kubernetes.slack.com/archives/CAR30FCJZ/p1604861145449700

c/c @erikgb